styleguide/derek/view-templates/stories/_stories.scss

```diff
@@ -1,3 +1,218 @@
+// Customer Stories Feature Colors
+$cust-stories-red: #9d0000;
+$cust-stories-teal: #00b4c3;
+$cust-stories-purple: #563777;
```

Comment:
We can't use the ones we have? Or lighten or darken the existing swatches? I'd hate to have these as hex values when they are so close to what we already have.